src/main/java/com/google/ehub/utility/NextIncreasingLexicographicStringUtility.java

```diff
@@ -0,0 +1,37 @@
+package com.google.ehub.utility;
+
+/**
+ * Utility class that holds method to find the next lexicographic string.
+ */
+public final class NextIncreasingLexicographicStringUtility {
```

Comment:
Having a whole class for one method is a little too specific I think. Usually we just have a Utils.java file with a couple random methods that could get reused. But is this maybe something only the dashboard will use? Maybe we can just have this as a static method there

Reply:
This class can be used for servlets that implement searches for any kind of object in Datastore. I wanted to keep it separate so the testing file only dealt with testing the algorithm that was implemented and not non-related Servlet tests.

Reply:
I see; hm, ok it's still usually not great practice to create a class for just a single util method. Looking back on how you use this, I think you actually could do something like:

```
class QueryUtils {

  // Move reusable code above in EIDatastore in here
  public static CompositeFilterOperator getPrefixFilter(String normalizedPrefix) {
    // blah blah blah... calls getNextHighestString(normalizedPrefix)
  }
  
 static String getNextHighestString(String str) {

  }

}
```
And now if we want Oyin to have a search for users feature, she can also just call getPrefixFilter(); what do you think about doing it this way?

Reply:
Yeah adding all this methods into a single Utility class makes more sense. I merged utility methods into a single class in last commit.